public/app/themes/justice/package.json

```diff
@@ -25,15 +25,15 @@
     "devDependencies": {
         "@sentry/integrations": "^7.114.0",
         "@tinypixelco/laravel-mix-wp-blocks": "1.2.0",
-        "@wordpress/babel-preset-default": "^7.36.0",
-        "@wordpress/data": "^9.28.0",
-        "@wordpress/dependency-extraction-webpack-plugin": "^5.2.0",
-        "@wordpress/icons": "^9.43.0",
-        "@wordpress/preferences": "^4.0.0",
-        "@wordpress/rich-text": "^7.0.0",
+        "@wordpress/babel-preset-default": "^6.17.0",
```

Comment:
Was this set to an earlier version on purpose? (7 -> 6)

Reply:
Hey, good spot. TY :) I've updated to 8